src/Products/PlonePAS/browser/info.py

```diff
@@ -7,6 +7,7 @@
 from Products.PluggableAuthService.interfaces.plugins import (  # noqa: E501
     ILoginPasswordExtractionPlugin,
 )
+from zExceptions import HTTPImATeapot
```

Comment:
:-D

For reference: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/418